UI/data/locale/en-US.ini

```diff
@@ -591,6 +591,14 @@ Basic.Main.AddSceneCollection.Text="Please enter the name of the scene collectio
 # rename scene collection dialog
 Basic.Main.RenameSceneCollection.Title="Rename Scene Collection"
 
+# remigrate scene collection dialog
+Basic.Main.RemigrateSceneCollection.Title="Update Collection Resolution"
+Basic.Main.RemigrateSceneCollection.Text="Do you want to update the scene collection resolution of "%1" to match the current profile's canvas resolution of %2x%3?"
```

Comment:
Do the inner quotes need to be replaced by single-quotes or escaped?
```
of "%1" to
```

Reply:
They need to be escaped, yes. This was a mistake introduced when updating the strings based on earlier feedback.